packages/toolpad-app/src/bindings.ts

```diff
@@ -0,0 +1,26 @@
+import { BindableAttrValue } from '@mui/toolpad-core';
+
+export function toBindable<V>(
```

Comment:
I'd like to avoid sprinkling these functions around the whole code base and essentially have two formats of bindings used together. In the appDom, can we change the interface of a binding from
```ts
{
  type: 'jsExpression'
  value: string
}
```
to the interface used by the file schema
```ts
{
  $$jsExpression: string
}
```
so that we have only one format to worry about in the whole codebase.

Reply:
Yeah, I wasn't sure we should make this change already, if so it sounds great!